gravitee-apim-cypress/package.json

```
@@ -6,10 +6,11 @@
     "scripts": {
         "test": "cypress open",
         "test:bulk": "cypress run --config-file cypress/integration/bulk/cypress-config.json",
-        "test:ci": "cypress run --config-file cypress/integration/nrt/cypress-config.json",
+        "test:ci": "cypress run --config-file cypress/integration/rest-api/cypress-config.json",
```

to keep consistency i would suggest:
```suggestion
        "test:rest-api": "cypress run --config-file cypress/integration/rest-api/cypress-config.json",
```

---

Alright, I'll adjust it.